hedera-mirror-rosetta/app/domain/types/operation.go

```diff
@@ -19,5 +21,21 @@ func (t *Operation) FromRosettaOperation(rOperation *rTypes.Operation) *Operatio
 }
 
 // ToRosettaOperation returns Rosetta type Operation from the current domain type Operation
-func (t *Operation) ToRosettaOperation(*rTypes.Operation) {
+func (t *Operation) ToRosettaOperation() *rTypes.Operation {
+	rOperation := rTypes.Operation{
+		OperationIdentifier: &rTypes.OperationIdentifier{
+			Index: t.Index,
+		},
+		RelatedOperations: []*rTypes.OperationIdentifier{}, //TODO
```

Comment:
I am not sure what I need TODO to finish it, can you add a bit more info on the TODO

Reply:
Also I am not sure if we will ever need to add Related Operation

Reply:
I just saw that it is not required by the API. However, my impressions (from the conversations we had) were that we would implement it.

Reply:
I think this has to do with the contract calls, etc. that we probably wont be using